app/src/components/SomeBuyPage.jsx

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
dont include in a PR if its supposed to be removed later, just make the initial component and leave it empty with a todo or something, otherwise it might be forgotten about and left